json/Arles_Text.txt

```diff
@@ -5812,215 +5812,215 @@
 		"jp_text": "……アーレスバトルはめくらましだ。<%br>ネストの真の目的は、別にある！",
-		"tr_text": "",
+		"tr_text": "...The Ares Battle was just a cover.<%br>Nest's after something much worse!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "バトル会場が見えてきたぞ！<%br>……ソードとブレードが戦ってる！",
-		"tr_text": "",
+		"tr_text": "We're nearly at the arena!<%br>...Sword and Blade are fighting!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "カイゼルは……いたわ！　まだ無事よ！",
-		"tr_text": "",
+		"tr_text": "Kaiser is... She's still there! We aren't safe yet!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "？？？",
 		"tr_name": "???",
 		"jp_text": "ふむ……ようやく役者が揃ったね。<%br>では、はじめようか。",
-		"tr_text": "",
+		"tr_text": "Hmm... The actors are finally assembled.<%br>Well then, let us begin.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "カイゼルさーん！<%br>すぐにバトルを中止してください！",
-		"tr_text": "",
+		"tr_text": "Kaiseeer!<%br>Please, stop the battle now!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "アーレスカイゼル",
 		"tr_name": "Ares Kaiser",
 		"jp_text": "ジェネ……？　どうしたのですか？",
-		"tr_text": "",
+		"tr_text": "Gene...? What has happened?",
 		"fileID": 2
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "アーレスネストさんの目的は……キャアァッ！",
-		"tr_text": "",
+		"tr_text": "Ares Nest is planning... Aaaaah!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "ば、爆撃！？　どこから……",
-		"tr_text": "",
+		"tr_text": "B-bombardment?! From where...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "うぅわぁぁぁっ！　あぶ、あぶないってば！",
-		"tr_text": "",
+		"tr_text": "Waaaaaah! We- we're in danger!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "もしかして、ネストの狙いはカイゼルだけでなく……<%br>この場にいる私たち全員なの！？",
-		"tr_text": "",
+		"tr_text": "What if Nest wasn't just targeting Kaiser...<%br>What if she was targeting all of us here?!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "アーレスパルチザン",
 		"tr_name": "Ares Partizan",
 		"jp_text": "な、なにが起きてるんだ！？<%br>とにかくカイゼル様を、守ら……ねば……",
-		"tr_text": "",
+		"tr_text": "W-what's going on?!<%br>We have to protect... Lady... Kaiser...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "アーレスブレード",
 		"tr_name": "Ares Blade",
 		"jp_text": "……この先には行かせない。<%br>ま、そのケガじゃ動けないだろうけど。",
-		"tr_text": "",
+		"tr_text": "...I won't let you leave this spot.<%br>You wouldn't get far with that injury, anyway.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "アーレスソード",
 		"tr_name": "Ares Sword",
 		"jp_text": "そこを……どけぇぇぇっ！！",
-		"tr_text": "",
+		"tr_text": "Get out... of my way!!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "アーレスブレード",
 		"tr_name": "Ares Blade",
 		"jp_text": "甘いっ！！",
-		"tr_text": "",
+		"tr_text": "Weak!!",
```

Comment:
Or this.

Reply:
Sweet!!

Reply:
Looking at the dictionary entry for it, I thought that might have been
`5. half-hearted; not finished properly​`
or
`6. insufficient; not satisfactory; inadequate; loose​`